autopush/utils.py

```diff
@@ -524,4 +587,10 @@ def websocket_format(self):
 def ms_time():
     # type: () -> int
     """Return current time.time call as ms and a Python int"""
-    return int(time.time() * 1000)
+    return int(time.time() * pow(10, 3))
```

Comment:
Was getting tired of counting zero's between 3 of them and 6. I've seen typo's creep in as well where instead of 6 there'll be 3. It is a pain to call them constantly though, so I'll refactor that into a const.